kraft-liveness-readiness.md

```diff
@@ -0,0 +1,112 @@
+# Liveness and Readiness probes in a KRaft Kafka cluster
```

Comment:
Should we file be named `43-kraft-liveness-readiness.md` ?

Reply:
Good spot, I've aligned the file names properly now